src/dev/GraphiQL.tsx

```diff
@@ -67,25 +67,42 @@ const httpFetcher: Fetcher = (graphQLParams) =>
 const httpMultipartFetcher: Fetcher = (graphQLParams) =>
   ({
     subscribe: (...args: SubscribeArguments) => {
+      const abortController = new AbortController();
       const sink = getSinkFromArgs(args);
       const isIntrospectionQuery = args.length === 3;
```

Comment:
whoops, think we can get rid of this
```suggestion
```

Reply:
Eh, this is for an inconsistency issue inside graphiql. See https://github.com/graphql/graphiql/pull/1639 and https://github.com/graphql/graphiql/issues/1638

Reply:
ah nervermind :D

Reply:
I applyied the change via commit suggestion :)